qemu/tests/uefi_check_secure_MOR.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import os
+
+from avocado.utils import process
+
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    Please make sure the guest installed with signed driver
+    Verify Secure MOR control feature using Device Guard tool in Windows guest:
+
+    1) Boot up a guest.
+    2) Check if Secure Boot is enable.
+    3) Download Device Guard and copy to guest.
+    4) Enable Device Guard and check the output.
+    5) Reboot guest.
+    5) Run Device Guard and check the output.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def execute_powershell_command(command):
+        status, output = session.cmd_status_output(command)
+        logging.info("command:%s" % command)
+        logging.info("command output:%s" % output)
+        if status != 0:
+            test.fail("execute command fail: %s" % output)
+        return output
+    timeout = int(params.get("timeout", 360))
+    params['ovmf_vars_filename'] = 'OVMF_VARS.secboot.fd'
+    params['start_vm'] = 'yes'
+    params['cpu_model_flags'] = ',hv-passthrough'
+    env_process.preprocess_vm(test, params, env, params['main_vm'])
```

Comment:
how about setting above 3 params in cfg (start_vm = yes), then we don't need to preprocess_vm, vm can be started automatically, we just log into VM and do our tests